ChaosMod/Effects/db/Misc/MiscFlashbang.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+	Effect by <Your Name>
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+	DWORD ticksToDraw = GetTickCount() + 3000;
+	int screenx;
+	int screeny;
+	GET_SCREEN_RESOLUTION(&screenx, &screeny);
+	while (GetTickCount() < ticksToDraw)
+	{
+		WAIT(0);
+		DRAW_RECT(0,0, screenx, screeny, 255, 255, 255, 255, 0);

```

Comment:
Iirc the width and height are percentages -> 1.f is 100% screen-width, no need to get the screen size